### PR TITLE
Add bsdmake

### PR DESCRIPTION
**Pull request by CRKatri**

### CODE REVIEW

**Comment by CRKatri** [comment]
Sorry, didn't mean to push the FreeBSD changes, I'll fix that.

**Comment by Diatrus** [comment]
Please implement libiosexec :)

**Comment by Diatrus** [comment]
Last thing: you don't need a patch for iosexec anymore: you just need a sed line. See libuv.

File: build_info/bsdmake.control [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Package: bsdmake
+Maintainer: @DEB_MAINTAINER@
+Architecture: @DEB_ARCH@
+Version: @DEB_BSDMAKE_V@
+Section: Development
+Depends: libiosexec1
```
Comment (asdfugil) [inline build_info/bsdmake.control:6]: There are no macOS libiosexec1

File: build_info/bsdmake.control [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Package: bsdmake
```
Comment (asdfugil) [inline build_info/bsdmake.control:1]: Debianize this 
https://packages.debian.org/sid/bmake

**Review by asdfugil — CHANGES_REQUESTED** [review]

File: build_info/bsdmake.control (hunk 2 above)
Comment (CRKatri) [reply build_info/bsdmake.control:1]: These are different packages, this is `bsdmake`, that is `bmake`, they are different implementations of BSD make.

**Review by asdfugil — APPROVED** [review]

**Pull request closed without merging**